frontend/mainwindow.cpp

```diff
@@ -77,6 +73,8 @@ void MainWindow::initialiseElements()
     ui_->runDataTable->setContextMenuPolicy(Qt::CustomContextMenu);
     connect(ui_->runDataTable, SIGNAL(customContextMenuRequested(QPoint)), SLOT(customMenuRequested(QPoint)));
     contextMenu_ = new QMenu("Context");
+
+    on_closeFind_clicked();
```

Comment:
Calling this causes a segfault as the `selectionModel()` in the run table doesn't yet exist (I think).

Also, if there's general widget dis/enabling to be done, it would be better to put that in its own independent function and have the `on_closeFind_clicked` slot call that instead.